service/controller/drainer.go

```diff
@@ -45,7 +48,7 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 				LabelSelector: fmt.Sprintf("%s=%s", key.PodWatcherLabel, config.ProjectName),
 			},
 			RateWait:     informer.DefaultRateWait,
-			ResyncPeriod: informer.DefaultResyncPeriod,
+			ResyncPeriod: 30 * time.Second,
```

Comment:
The drainer is short lived like in the aws-operator. We check the pods all 30 seconds. Works nice.